assets/src/css/codefund-framework/_css-loader.scss

```diff
@@ -0,0 +1,101 @@
+.cssload-wrapper {
+  padding-left: 1em;
+  padding-right: 1em;
+  margin: auto;
+  display: block;
+  width: 195px;
+}
+
+.cssload-loader {
+  width: 49px;
+  height: 49px;
+  border-radius: 50%;
+  margin: 3em;
+  display: inline-block;
+  position: relative;
+  vertical-align: middle;
+  background: #0ece7c;
+}
+
+.cssload-loader,
+.cssload-loader:before,
+.cssload-loader:after {
+  animation: 2.5s infinite ease-in-out;
+  -o-animation: 2.5s infinite ease-in-out;
+  -ms-animation: 2.5s infinite ease-in-out;
+  -webkit-animation: 2.5s infinite ease-in-out;
+  -moz-animation: 2.5s infinite ease-in-out;
+}
+
+.cssload-loader:before,
+.cssload-loader:after {
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.cssload-loader {
+  animation-name: cssload-loader;
+  -o-animation-name: cssload-loader;
+  -ms-animation-name: cssload-loader;
+  -webkit-animation-name: cssload-loader;
+  -moz-animation-name: cssload-loader;
+}
+
+@keyframes cssload-loader {
+  from {
```

Comment:
can we put these from/to styles in a class and use them in these functions

Reply:
I just copied/pasted from another site. I'd rather not refactor unless you insist.

Reply:
eh not super critical